node/forest_libp2p/src/chain_exchange/behaviour.rs

```diff
@@ -64,7 +67,11 @@ impl ChainExchangeBehaviour {
         self.track_metrics();
         if let Some(tx) = self.response_channels.remove(request_id) {
             if let Err(err) = tx.send(Err(error.into())) {
-                warn!("{err}");
+                // Demoting log level here because the same request might be sent to multiple
+                // remote peers simultaneously, it's expected that outbound failures that happen
+                // after receiving the first succussful response could be sent to a closed
+                // channel.
```

Comment:
```suggestion
                // Demoting log level here because the same request might be sent to multiple
                // remote peers simultaneously, it's expected that outbound failures that happen
                // after receiving the first successful response could be sent to a closed
                // channel.
```